theme_zap/views/snippets/s_references.xml

```diff
@@ -4,45 +4,47 @@
 <template id="s_references" inherit_id="website.s_references">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pb64 o_cc2" remove="o_cc1 pb80" separator=" "/>
+        <attribute name="class" add="o_cc4" remove="o_cc1" separator=" "/>
     </xpath>
     <!-- Title -->
     <xpath expr="//h2" position="replace" mode="inner">
-        Our <b>References</b>
+        Our References
     </xpath>
     <!-- Paragraph -->
-    <xpath expr="//p" position="after">
-        <p><br/></p>
+    <xpath expr="//p[hasclass('lead')]" position="replace" mode="inner">
+        We're excited to have the opportunity to collaborate with these outstanding companies!
     </xpath>
+    <!-- Link -->
+    <xpath expr="//a" position="replace"/>
     <!-- Column #01 -->
     <xpath expr="//div[hasclass('col-lg-2')][1]" position="attributes">
-        <attribute name="class" add="o_cc o_cc1 rounded pt48 pb48" remove="pt16 pb16" separator=" "/>
-        <attribute name="style" add="box-shadow: rgba(0, 0, 0, 0.15) 0px 2px 4px 0px !important;" separator=" "/>
+        <attribute name="class" add="o_cc o_cc1 rounded pt48 pb48 border" remove="pt16 pb16" separator=" "/>
+        <attribute name="style" add="border-width: 4px !important; border-color: var(--o-color-1) !important;" separator=" "/>
```

Comment:
```suggestion
        <attribute name="style" add="border-width: 4px !important; border-color: var(--o-color-1) !important;" separator=";"/>
```

Reply:
Are you sure ? As i'm using the ";" in my style declaration, I think the logical separator is " ", no ? It works perfectly fine how I did and it is also cleaner:

`separator=" "` : `style="border-width: 4px !important;border-color: var(--o-color-1) !important"`
`separator=";"` : `style="border-width: 4px !important; border-color: var(--o-color-1) !important;"`

Reply:
Yes, because if the style attribute you extend changes (or is extended by another template), it could/would break.
Additionally, this is a rule that's about to be enforced by https://github.com/odoo/design-themes/pull/933

Reply:
Okay, makes sense. All fixed 👍